wuvt/static/css/wuvt.css

```diff
@@ -794,4 +811,94 @@ body > footer {
     font-size: 0.7em;
     text-align: center;
 }
+
+@media only screen and (max-width: 810px) {
```

Comment:
all the px-heavy CSS is going to be brittle in the future (and will possibly have scaling issues on high-resolution screens). consider modifying.